nx/include/switch/services/lbl.h

```diff
@@ -17,15 +24,62 @@ void lblExit(void);
 /// Gets the Service object for the actual lbl service session.
 Service* lblGetServiceSession(void);
 
-Result lblSwitchBacklightOn(u64 fade_time);
-Result lblSwitchBacklightOff(u64 fade_time);
+Result lblSaveCurrentSetting(void);
+Result lblLoadCurrentSetting(void);
 
 /**
  * @note The brightness goes from 0 to 1.0.
  */
 Result lblSetCurrentBrightnessSetting(float brightness);
 Result lblGetCurrentBrightnessSetting(float *out_value);
 
+Result lblApplyCurrentBrightnessSettingToBacklight(void);
+Result lblGetBrightnessSettingAppliedToBacklight(float *out_value);
+
+Result lblSwitchBacklightOn(u64 fade_time);
+Result lblSwitchBacklightOff(u64 fade_time);
+Result lblGetBacklightSwitchStatus(LblBacklightSwitchStatus *out_value);
+
+Result lblEnableDimming(void);
+Result lblDisableDimming(void);
+Result lblIsDimmingEnabled(bool *out_value);
+
 Result lblEnableAutoBrightnessControl(void);
 Result lblDisableAutoBrightnessControl(void);
 Result lblIsAutoBrightnessControlEnabled(bool *out_value);
+
+Result lblSetAmbientLightSensorValue(float value);
+Result lblGetAmbientLightSensorValue(bool *over_limit, float *lux);
```

Comment:
applet uses this and lblIsAmbientLightSensorAvailable, add that to these docs (see wiki).